modules/http/http.c

```diff
@@ -223,6 +223,22 @@ http_dd_set_timeout(LogDriver *d, glong timeout)
   self->timeout = timeout;
 }
 
+void
+http_dd_set_flush_lines(LogDriver *d, glong flush_lines)
```

Comment:
I've addressed this in 882d032510b4d8368e0b16c42bff7a2d2d566597

With that, could you please approve the patchset? The rest of the changes requested were added to a subsequent PR, as those would cause massive conflicts. Thanks.